src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,36 @@
-export const TodoFilter = () => (
+interface Props {
+  value: string;
+  changeFilter: (string: string) => void;
+  changeInput: (string: string) => void;
+}
+
+export const TodoFilter: React.FC<Props> = ({
+  value,
+  changeFilter,
+  changeInput,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select
+          data-cy="statusSelect"
+          onChange={(event) => changeFilter(event.target.value)}
+        >
+          <option
+            value="all"
+          >
+            All
+          </option>
+          <option
+            value="active"
+          >
+            Active
+          </option>
+          <option
+            value="completed"
+          >
+            Completed
+          </option>
```

Comment:
will be better iterate `option` using `.map()` and `enum` for value

Reply:
@Stanislav Sokolov  "will be better iterate option using .map() and enum for value"


If I recall, enum has the behaviour of an object, while .map() is an array method. Do you propose to make something like this?


```
enum ListOfOptions {
  array = [
    {title: 'All', value: 'all'},
    {title: 'Active', value: 'active'},
    {title: 'Completed', value: 'completed'},
  ]
}

ListOfOptions.array.map(option => (
  <option
    value=option.value
  >
    {option.title}
  </option>
))
```


Because this looks EXTREMELY cumbersome